handler/handler.go

```diff
@@ -138,22 +161,90 @@ func DeleteCandidate(res http.ResponseWriter, req *http.Request) {
 // Requires admin privileges
 // Endpoint: GET /api/voters
 func GetValidVoters(res http.ResponseWriter, req *http.Request) {
-
+	if !isAuthenticatedWithPermissions(res, req, []string{"viewAdmin"}) {
+		return
+	}
+
+	voters, err := db.QueryAllVoters()
+	if err != nil {
+		util.RequestError(req.Context(), res, util.ServerError(err.Error()))
+		return
+	}
+
+	var response_data struct {
+		Voters []models.Voter
+	}
+
+	response_data.Voters = voters
+	err = util.WriteJson(res, response_data)
+	if err != nil {
+		util.RequestError(req.Context(), res, err)
+		return
+	}
 }
 
 // AddValidVoters adds a list of users(emails) to the list of valid voters
 // Duplicates of users already in the system will be ignored
 // Requires admin privileges
 // Endpoint: PUT /api/voters/add
 func AddValidVoters(res http.ResponseWriter, req *http.Request) {
-
+	if !isAuthenticatedWithPermissions(res, req, []string{"modifyAdmin"}) {
+		return
```

Comment:
I would prefer permission names to be defined in some type of util file. 
```suggestion
MODIFY_ADMIN = "modifyAdmin"
...
if !isAuthenticatedWithPermissions(res, req, []string{MODIFY_ADMIN}) {
```

Reply:
:+1: